### ISSUE TITLE
PHP Fatal error:  Class 'Terminus\Models\Collections\Sites' not found

### ISSUE THREAD

**Issue by dsnopek** [rvtraveller/terminus-omniscient#2]
I just installed the plugin, using the latest version of Terminus (0.13.3) and got the following error when trying to run `terminus sites omniscient`:

```
PHP Fatal error:  Class 'Terminus\Models\Collections\Sites' not found in /home/dsnopek/terminus/plugins/terminus-omniscient/Commands/OmniscientCommand.php on line 26
PHP Stack trace:
PHP   1. {main}() /home/dsnopek/terminus/vendor/pantheon-systems/terminus/php/boot-fs.php:0
PHP   2. include() /home/dsnopek/terminus/vendor/pantheon-systems/terminus/php/boot-fs.php:25
PHP   3. Terminus\Runner->run() /home/dsnopek/terminus/vendor/pantheon-systems/terminus/php/boot-cl.php:23
PHP   4. Terminus\Runner->runCommand() /home/dsnopek/terminus/vendor/pantheon-systems/terminus/php/Terminus/Runner.php:212
PHP   5. Terminus\Dispatcher\Subcommand->invoke() /home/dsnopek/terminus/vendor/pantheon-systems/terminus/php/Terminus/Runner.php:309
PHP   6. call_user_func() /home/dsnopek/terminus/vendor/pantheon-systems/terminus/php/Terminus/Dispatcher/Subcommand.php:121
PHP   7. Terminus\Dispatcher\CommandFactory::Terminus\Dispatcher\{closure}() /home/dsnopek/terminus/vendor/pantheon-systems/terminus/php/Terminus/Dispatcher/Subcommand.php:121
PHP   8. Terminus\Commands\OmniscientCommand->__construct() /home/dsnopek/terminus/vendor/pantheon-systems/terminus/php/Terminus/Dispatcher/CommandFactory.php:131
```

Thanks in advance for your help!

**Comment by rvtraveller** [comment]
Please try updating to the latest version.  This should be resolved now that #1 is merged..

**Comment by kzalewski** [comment]
I'm running Terminus 0.13.6, and this error is happening for me as well.  It was not happening when using an earlier release in the 0.13 branch.

The problem is that Pantheon developers moved the Sites class.

In the plugin code, try changing the class path in the "use" statement from Terminus\Models\Collections\Sites to Terminus\Collections\Sites.  That should resolve the issue.
